tutorials/tour/basics.md

```diff
@@ -37,51 +37,52 @@ println("Hello," + " world!") // Hello, world!
 
 ### Values
 
-You can name results of expressions by using the `val` keyword.
+You can name results of expressions with the `val` keyword.
 
-```
+```tut
 val x = 1 + 1
-println(x) // 2
+println(x * x) // 4
```

Comment:
I see.

How about leaving this line as is and adding another example, maybe something like below?

> You can use values in other expressions.
> ```
> val x = 1 + 1
> println(x * 2) // 4
> ```